scripts/manage-db.py

```diff
@@ -13,22 +14,74 @@
 
 from auslib.db import AUSDatabase
 
+def cleanup_releases(trans, nightly_age, dryrun=True):
+    # This and the subsequent queries use "%%%%%" because we end up going
+    # through two levels of Python string formatting. The first is here,
+    # and the second happens at a low level of SQLAlchemy when the transaction
+    # is being executed.
+    query = """
+LEFT OUTER JOIN releases_history USING (name)
+LEFT JOIN rules ON (name=mapping)
+WHERE name LIKE '%%%%nightly%%%%'
+AND (rules.whitelist <> releases.name OR rules.whitelist IS NULL)
+AND (rules.mapping <> releases.name OR rules.mapping IS NULL)
+AND (timestamp<1000*UNIX_TIMESTAMP(NOW()-INTERVAL %s MONTH) OR change_id is NULL);
+""" % nightly_age
+    if dryrun:
+        todelete = trans.execute("SELECT name FROM releases" + query).fetchall()
```

Comment:
The example code isn't the best example. I like seeing the fields used in the WHERE condition to see how the sql engine matched things. Since there is an `AND ( a OR b )` condition, it would help to see how that part matched.